### ISSUE TITLE
Cheat mode click not working like it used to

### ISSUE THREAD

**Issue by RepoDraghon** [mezz/JustEnoughItems#1487]
Recently updated ALL of the mods in the Direwolf20 v2.4.0 pack+some other mods to see if it works with all the latest mods.
It does, however have 2 issues, 1 has nothing to do with JEI/NEI

1st issue, can't sleep, game hangs and I have to force crash the game. (Not your issue)

2nd issue, when looking to get 64 or just 1 more of an item I already have, I usually press Ctrl.+left (for total 64) or right (for only 1 more of item) click is now not working. If I now Ctrl+middle click, it will now give me a total of 64 of that item I clicked on in my inventory. How to fix it the old way?

**Comment by mezz** [comment]
Thanks for the report!

I think you explained a few things but it is not immediately clear what you are reporting.
Please try reporting in this style:

 * The single action you took. (shift+clicked a specific button)
 * The result you expect. (infinite riches)
 * The result you got instead. (1 dirt block)

If you have more than one buggy action with more than one result, please break them up into multiple groups like that.

**Comment by RepoDraghon** [comment]
sorry.

1. press ctrl+left click on item in actual inventory
2. expect total of 64 of clicked item
3. result 1 item moved from inventory to hotbar

https://youtu.be/5XXBzfYUI3c

Suspicion that something has changed the key mapping as I have tried downgrading to v 222 but nothing was fixed and there's no manual key mapping at all...

Confirmed, reverted to ORIGINAL Direwolf20 pack (Not fully updated mods) plus all of the mods I added. The keybinds are working properly now. Now resisting urge to update mods, lol
